### PR TITLE
product_usability: seller_id can not be store as muti-company will be broken

### PR DESCRIPTION
**Pull request by sebastienbeau**
In multi company each company can have it own supplier with shared product, in that case supplier id are different
https://github.com/akretion/odoo-usability/issues/211

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@alexis-via

**Review by bealdav — APPROVED** [review]

**Comment by bealdav** [comment]
Good catch

**Comment by bealdav** [comment]
Meanwhile in the world of oca

https://github.com/OCA/product-attribute/tree/14.0/product_main_supplierinfo
https://github.com/OCA/product-attribute/tree/16.0/product_main_supplierinfo
https://github.com/OCA/product-attribute/pull/1810

File: product_usability/models/product_template.py [hunk 1]
```diff
@@ -33,6 +35,19 @@ class ProductTemplate(models.Model):
     company_id = fields.Many2one(tracking=110)
     barcode_type = fields.Char(compute='_compute_template_barcode_type')
 
+
+    def _search_seller_id(self, operator, value):
+        # searching on the first line of a o2m is not that easy
+        # So we search all potential matching products
+        # Then we filter on the seller_id
+        records = self.search([("seller_ids.partner_id", operator, value)])
```
Comment (alexis-via) [inline product_usability/models/product_template.py:43]: BOUM ! CRASH !
seller_ids.name and not seller_ids.partner_id

**Pull request merged**